go/launchd/launchd.go

```diff
@@ -128,6 +128,28 @@ func (s Service) Restart() error {
 	return Restart(s.Label(), s.log)
 }
 
+// WaitForStatus waits for service status to be available
+func (s Service) WaitForStatus(wait time.Duration) (*ServiceStatus, error) {
+	t := time.Now()
+	i := 1
+	for time.Now().Sub(t) < wait {
+		status, err := s.LoadStatus()
+		if err != nil {
+			return nil, err
+		}
+		if status != nil {
+			return status, nil
+		}
+		// Tell user we're waiting for status after 4 seconds, every 4 seconds
```

Comment:
`LoadStatus` can return `nil,nil` if control falls off the end of the function, i..e, if `found` isn't `true`.  Is this what happened?